code/game/machinery/iv_drip.dm

```diff
@@ -145,6 +145,8 @@
 				update_icon()
 
 /obj/machinery/iv_drip/attack_hand(mob/user)
+	if (!ishuman(user))
```

Comment:
I'd say let them do it, until someone codes a proper IV inserter item for the medical borg